src/index.js

```diff
@@ -1,76 +1,164 @@
+import Dexie from 'dexie';
 import UnityCacheError from './error';
-import localforage from 'localforage/dist/localforage.nopromises';
 
-const DB_DRIVER = [ localforage.INDEXEDDB, localforage.WEBSQL, localforage.LOCALSTORAGE ];
 const RE_BIN = /^\w+$/;
 const EXPIRE_BIN = '___expire___';
 const EXPIRE_GLUE = '::';
 const DEFAULT_NAME = 'unity';
-const DEFAULT_DESCRIPTION = 'Unity cache';
+const DEFAULT_VERSION = 1;
 
-let cacheBins = {};
+const cacheInstance = {
+    config: {},
+    db: null
+};
 
-function createCacheBins(bins, name, description, driver) {
-    bins = [].concat(bins, EXPIRE_BIN);
+function setCacheConfig(name, stores, version) {
+    stores = [].concat(stores, EXPIRE_BIN);
 
-    return bins.reduce((result, storeName) => {
+    stores = stores.reduce((result, storeName) => {
         if (!RE_BIN.test(storeName)) {
             throw new UnityCacheError(`Store names can only be alphanumeric, '${storeName}' given`);
         }
 
-        result[storeName] = localforage.createInstance({
-            storeName,
-            name,
-            description,
-            driver
-        });
-
+        result[storeName] = '&';
```

Comment:
For each store we need pairs `[key, value]` (field key without name). `&` means that keys must be unique.